browser/src/control/jsdialog/Util.Dropdown.js

```diff
@@ -168,6 +170,14 @@ JSDialog.OpenDropdown = function (id, popupParent, entries, innerCallback, popup
 	};
 	L.Map.THIS.fire('closepopups'); // close popups if a dropdown menu is opened
 	L.Map.THIS.fire('jsdialog', {data: json, callback: generateCallback(entries)});
+
+	var dropDialog = L.Map.THIS.jsdialog.getDialog(dropdownWindowId);
+	if (dropDialog && dropDialog.container) {
+		var container = dropDialog.container.querySelector('#' + id + '-entries');
+		if (container) {
+			container.role = 'listbox';
```

Comment:
Can we setup this role on that element when it is created?
https://github.com/CollaboraOnline/online/blob/ae8300e5f556ce914e2c9256b6b2276a53520678/browser/src/control/jsdialog/Util.Dropdown.js#L35

We could add "accessibilityRole: 'listbox'" to the JSON.
Then it can be applied in postprocessing of every widget: https://github.com/CollaboraOnline/online/blob/ae8300e5f556ce914e2c9256b6b2276a53520678/browser/src/control/Control.JSDialogBuilder.js#L2835

This way you don't need to write code for every case.

Reply:
I have tried to follow the recursive builder algorithm, which populates a container in the general case, based on the JSON data provided here. Yes, I can inject the "accessibilityRole" field into the combobox, assuming it is the data is re-created in the combobox widget.